### PR TITLE
Remove deprecations in v0.38.0 to remove at v0.43.0

### PR DESCRIPTION
**Pull request by tkoyama010**
### Overview

Remove deprecations in v0.38.0 to remove at v0.43.0.

### Details

- None

### CODE REVIEW

File: pyvista/plotting/widgets.py [hunk 1]
```diff
@@ -1310,10 +1310,6 @@ def add_text_slider_widget(
         **kwargs : dict, optional
             Deprecated keyword arguments.
```
Comment (dcbr) [inline pyvista/plotting/widgets.py:1311]:
```suggestion
```
I guess this can also be removed now?

File: pyvista/plotting/widgets.py [hunk 2]
```diff
@@ -1465,10 +1461,6 @@ def add_slider_widget(
...
```
Comment (dcbr) [inline pyvista/plotting/widgets.py:1463]:
```suggestion
```
I guess this can also be removed now?

File: pyvista/plotting/widgets.py (hunk 1 above)
Comment (tkoyama010) [reply pyvista/plotting/widgets.py:1311]: Good catch! I didn't notice that.

File: pyvista/plotting/widgets.py (hunk 2 above)
Comment (tkoyama010) [reply pyvista/plotting/widgets.py:1463]: Good catch! I didn't notice that.

**Review by dcbr — APPROVED** [review]
LGTM

**Pull request merged**